1.5/Source/PawnStorages/PawnStorages/PawnStorages.csproj

```diff
@@ -48,7 +48,7 @@
         </Reference>
     </ItemGroup>
 
-    <ItemGroup Condition="'@(Reference->WithMetadataValue('Identity', 'Assembly-CSharp')->Metadata('HintPath')->Exists())' == ''">
+    <ItemGroup Condition="'@(Reference-&gt;WithMetadataValue('Identity', 'Assembly-CSharp')-&gt;Metadata('HintPath')-&gt;Exists())' == ''">
```

Comment:
accidental I assume... wish it wouldn't keep doing this. Does it for me too.

Reply:
Fixed. 

It keeps bloody deleting PawnStorages.VEF.dll for me. Damned file refuses to exist on my system.

Reply:
Are you maybe building just the csproj instead of the solution? The `Homebound` csproj is the one that supposedly has the build chain set up so that one has a project ref to the other two as a sort of coordinator.